### ISSUE TITLE
Development discontinued? 

### ISSUE THREAD

**Issue by nimashariatian** [spree-contrib/spree_affiliate#16]
has the development on this extension been discontinued ?

**Comment by futhr** [comment]
No, we just waiting for [Store Credits](https://github.com/spree/spree/pull/5917) to get into Spree Core before updating the `master` branch. You welcome to submit PR's to other branches.

**Comment by nimashariatian** [comment]
perfect, thank you! does this work with 2.3 ?
